lib/providers/definitionsProvider.js

```diff
@@ -3,7 +3,7 @@
 import fs from 'fs';
 import path from 'path';
 import find from 'find';
-import mkdirp from 'mkdirp';
+import mkdirSync from 'mkdirp';
```

Comment:
I can't get the definition provider to work but based on the mkdirp docs it doesn't seem to provide a default export so I think it should be the below?

```suggestion
import { mkdirpSync } from 'mkdirp';
```